ciprs_reader/parser/section/header.py

```diff
@@ -13,7 +13,7 @@ class CaseDetails(HeaderParser):
     """Extract County and File No from header on top of first page"""
 
     pattern = (
-        r"\s*Case (Details|Summary) for Court Case[\s:]+(?P<county>(\w\s*)+) (?P<fileno>\w+)"
+        r"\s*Case (Details|Summary) for Court Case[\s:]+(?P<county>(\w\s*.*)+) (?P<fileno>\w+)"
```

Comment:
@copelco are we sure about using `.*` here? This regex looks good for the testcases we have and the generally expected output, but I'm curious if you've run into edge cases in the wild where unwanted characters would be matched.

If we definitely want to pull in all characters (including potential whitespace after the county), then this could simplify the regex a bit:

```
\s*Case (Details|Summary) for Court Case[\s:]+(?P<county>.+) (?P<fileno>\w+)
```

Reply:
@SherinV yes, thanks for your contribution!

@robert-w-gries Yes I think ``.*`` should be fine. I like your suggestion.